docs/ops/shape/Squeeze_1.md

```diff
@@ -4,29 +4,33 @@
 
 **Category**: Shape manipulation
 
-**Short description**: *Squeeze* removes specified dimensions (second input) equal to 1 of the first input tensor. If the second input is omitted then all dimensions equal to 1 are removed. If the specified dimension is not equal to one then error is raised.
+**Short description**: *Squeeze* removes dimensions equal to 1 from the first input tensor.
+
+**Detailed description**: *Squeeze* can be used with or without the second input tensor.
+*       If only the first input is provided, every dimension that is equal to 1 will be removed from it.
```

Comment:
Why 8 spaces here?

Reply:
I got confused about the spacing here. Changed it to one space now.